src/main/java/core/basesyntax/Triangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
```

Comment:
According to the task checklist, class/interface names should not use verbs. Consider renaming 'RightTriangle' to something that does not start with a verb, like 'Triangle' if it's a general triangle, or maintaining 'RightTriangle' if it specifically represents a right-angled triangle.